slides/src/content/angular-cli.md

```diff
@@ -0,0 +1,56 @@
+# Using the Angular CLI
+
+---
+
+## What is Angular-cli
+
+`Angular-cli` is a command line tool based on the `Ember-cli`
+to help scaffold and generate code for Angular applications
```

Comment:
I think it would be easier to read if we use bullets like this:

- `angular-cli` is a command line tool base on the `ember-cli`
- It's used to scaffold and generate code for Angular applications